includes/UserAdd.php

```diff
@@ -147,11 +154,18 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
-		echo "Error creating database for user, contact Admin";
+		
+		if($flag == 1)
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->